api/can_api_definition.py

```diff
@@ -59,6 +61,20 @@ class ResourceUtilization(base_model.APIBaseModel):
     )
 
 
+class Metrics(base_model.APIBaseModel):
```

Comment:
Can you add a class level docstring? If you pull in master, there are a few more examples.  The docstring ends up showing up in [the documentation](https://github.com/covid-projections/covid-data-model/blob/master/api/README.V1.md#resourceusageprojection)